scipy/stats/tests/test_morestats.py

```diff
@@ -835,6 +839,56 @@ def test_1d_input(self):
         assert_raises(ValueError, stats.levene, g1, x)
 
 
+class TestFEqualityTest:
+
+    def test_data1(self):
+        args = [f1, f2]
+        T, pval = f_equality_test(*args, alternative='two-sided')
+        assert_almost_equal(T, 1.207788039, 7)
+        assert_almost_equal(pval, 0.7105517535985708, 7)
+
+    def test_data2(self):
+        args = [f1, f2]
+        T, pval = f_equality_test(*args, alternative='greater')
+        assert_almost_equal(T, 1.207788039, 7)
+        assert_almost_equal(pval, 0.6447241232007151, 7)
+
+    def test_data3(self):
+        args = [f1, f2]
+        T, pval = f_equality_test(*args, alternative='less')
+        assert_almost_equal(T, 1.207788039, 7)
+        assert_almost_equal(pval, 0.3552758767992854, 7)
+
+    def test_data4(self):
+        args = [f1, f2]
+        T, pval = f_equality_test(*args)
+        assert_almost_equal(T, 1.207788039, 7)
+        assert_almost_equal(pval, 0.7105517535985708, 7)
+
+    def test_bad_arg(self):
+        args = [f1, f2]
+        # Too few args raises ValueError.
+        assert_raises(ValueError, f_equality_test, [1])
+        # Alternative not valid raises ValueError.
+        assert_raises(ValueError, f_equality_test, *args, 'twosided')
+        assert_raises(ValueError, f_equality_test, *args, 'lower')
+
+    def test_result_attributes(self):
+        args = [f1, f2]
+        res = f_equality_test(*args)
+        attributes = ('statistic', 'pvalue')
+        check_named_results(res, attributes)
+
```

Comment:
```suggestion
```
New hypothesis tests are supposed to return objects that cannot be unpacked. (I tried giving the author of a PR the liberty of using a named tuple recently, and others were very vocal against it.) The user (and your tests) should explicitly access the `statistic` and `pvalue` attributes.  Usually that means something like:

```
@dataclass
class AlexanderGovernResult:
    statistic: float
    pvalue: float
```
Go ahead and put something like that in here, but before merge I will replace it with a standard one that we use in many tests.